test/util/e2e.go

```diff
@@ -65,11 +65,19 @@ func CreateVisibilityClient(user string) visibilityv1alpha1.VisibilityV1alpha1In
 	return visibilityClient
 }
 
-func KueueReadyForTesting(ctx context.Context, client client.Client) {
+func KueueReadyForTesting(ctx context.Context, c client.Client) {
 	// To verify that webhooks are ready, let's create a simple resourceflavor
 	resourceKueue := utiltesting.MakeResourceFlavor("default").Obj()
-	gomega.Eventually(func() error {
-		return client.Create(context.Background(), resourceKueue)
+	gomega.EventuallyWithOffset(1, func() error {
+		return c.Create(ctx, resourceKueue)
 	}, StartUpTimeout, Interval).Should(gomega.Succeed())
-	ExpectResourceFlavorToBeDeleted(ctx, client, resourceKueue, true)
```

Comment:
why change this?

Reply:
the reporting is hard to follow with that,  in case of multikueue you don't know which of the clusters is not ready.